apps/web-giddh/src/app/shared/header/components/account-update-new-details/account-update-new-details.component.ts

```diff
@@ -31,8 +31,9 @@ import { IGroupsWithAccounts } from 'apps/web-giddh/src/app/models/interfaces/gr
 import { IFlattenGroupsAccountsDetail } from 'apps/web-giddh/src/app/models/interfaces/flattenGroupsAccountsDetail.interface';
 import { DbService } from 'apps/web-giddh/src/app/services/db.service';
 import { IDiscountList } from 'apps/web-giddh/src/app/models/api-models/SettingsDiscount';
-import { AssignDiscountRequestForAccount } from 'apps/web-giddh/src/app/models/api-models/ApplyDiscount';
+import { AssignDiscountRequestForAccount, ApplyDiscountRequestV2 } from 'apps/web-giddh/src/app/models/api-models/ApplyDiscount';
 import { SettingsDiscountActions } from 'apps/web-giddh/src/app/actions/settings/discount/settings.discount.action';
+import { element } from '@angular/core/src/render3';
```

Comment:
Avoid using nested imports
Please check if it is used

Reply:
yes it is used